src/CoreBundle/private/modules/MTTableManager/MTTableManager.class.php

```diff
@@ -477,56 +478,67 @@ protected function getAutocompleteRecords(): array
 
     private function getAutocompleteListQuery(): string
     {
-        $inputFilterUtil = $this->getInputFilterUtil();
-        $searchKey = $inputFilterUtil->getFilteredInput('term');
+        $databaseConnection = $this->getDatabaseConnection();
 
         $listClass = null;
         // allow custom list class overwriting (defined in pagedef)
         if (array_key_exists('listClass', $this->aModuleConfig)) {
             $listClass = $this->aModuleConfig['listClass'];
         }
 
-        $oTableList = &$this->oTableConf->GetListObject($listClass);
+        $listManager = &$this->oTableConf->GetListObject($listClass);
 
-        $query = $oTableList->FilterQuery();
-        $parentTableAlias = $oTableList->GetTableAlias($query);
+        $query = $listManager->FilterQuery();
 
-        $nameColumn = $this->oTableConf->GetNameColumn();
-        $autoClassName = TCMSTableToClass::GetClassName(TCMSTableToClass::PREFIX_CLASS, $this->oTableConf->fieldName);
-        $fieldIsTranslatable = call_user_func(array($autoClassName, 'CMSFieldIsTranslated'), $nameColumn);
+        $quotedNameToQuery = $this->getNameToQueryQuoted($listManager, $query);
 
-        if ($fieldIsTranslatable) {
-            $currentEditLanguageId = TdbCmsUser::GetActiveUser()->GetCurrentEditLanguageID();
-            $languageSuffix = TGlobal::GetLanguagePrefix($currentEditLanguageId);
-            if ('' !== $languageSuffix) {
-                $nameColumn .= '__'.$languageSuffix;
-            }
-        }
-        $databaseConnection = $this->getDatabaseConnection();
-        $quotedNameColumn = $databaseConnection->quoteIdentifier($nameColumn);
+        $searchTerm = '%'.$this->getInputFilterUtil()->getFilteredInput('term').'%';
+        $quotedSearchTerm = $databaseConnection->quote($searchTerm);
 
-        $quoteCharacter = $databaseConnection->getDatabasePlatform()->getIdentifierQuoteCharacter();
-        if (false === strpos($parentTableAlias, $quoteCharacter)) {
-            $quotedParentTableAlias = $databaseConnection->quoteIdentifier($parentTableAlias);
+        if (false !== stripos($query, 'WHERE')) {
+            // TODO / NOTE if $query contains an OR (without brackets) then this here will make the query wrong.
```

Comment:
So, this is an open issue that needs to be addressed? Any idea, how?

Reply:
No i see it. It was always a problem here. Because it is only a refactoring here we don`t need to address this issue at this time.